server/Database/database.js

```diff
@@ -0,0 +1,55 @@
+require('dotenv').config();
+const mongoose = require('mongoose');
+const dbUrl = process.env.MONGO_DB_URL;
+const localHost = process.env.LOCAL_HOST_URL;
+
+class Database
+{
+    constructor()
+    {
+        const url = process.env.IN_DEV === 'true'? localHost: dbUrl;

```

Comment:
Do you need all these extra checks? When you are primarily only seeking one url?
And if you are only seeking one url, why not just have one env variable that you set differently when you run localhost and when you run production?

Reply:
You make a valid point, changing env variables instead of the extra checks serves the same purpose.